libs/network/src/muddle/router.cpp

```diff
@@ -73,9 +73,11 @@ std::size_t GenerateEchoId(Packet const &packet)
   hash.Update(reinterpret_cast<uint8_t const *>(&channel), sizeof(channel));
   hash.Update(reinterpret_cast<uint8_t const *>(&counter), sizeof(counter));
 
-  auto const res = hash.Final();
+  std::size_t out = 0;
 
-  std::size_t const out = *reinterpret_cast<std::size_t const *>(res.pointer());
+  static_assert(sizeof(decltype(out)) == decltype(hash)::size_in_bytes,
```

Comment:
Pedantic: think it would read easier like: `static_assert(sizeof(out) == hash.size_in_bytes, "<message>");`

Reply:
I think clang-tidy may object to accessing static members through an instance, but I'll give it a go